docs/installation-and-operation/remote-sync.md

```diff
@@ -0,0 +1,347 @@
+   - **Move existing content:** Drag and drop items from other collections into the Library collection, or use the move option in the item's menu. 
+
+[Items in synced collections can't depend on items outside of synced collections](#items-in-synced-collections-cant-depend-on-items-outside-of-the-synced-collection). For example, if you try to add a question that references a model, make sure the model is also in a synced collection. 
+
+## 6. Push your changes to your repository
+
+Once you've added content, you'll see a yellow dot on your Library collection indicating uncommitted changes.
+
+1. Click the up arrow (push) icon next to the Library collection in the left sidebar.
+
+2. Enter a commit message describing your changes (e.g., "Added dashboard on mammoth populations").
+
+3. Click "Continue" to commit and push your changes to your repository.
+
+Check your repo, you should see the collection with your content in it.
+
+## 7. Create a personal access token for production
+
+Now that you have content in your repository, you can set up your production Metabase to pull that content.
+
+[Create a personal access token](#2-create-a-personal-access-token-for-development) following the same steps as before, but add Contents permissions to the token that are **Read-only** (NOT write), as you only want your production Metabase reading from the repo. (Contents permissions require Metadata permissions, which GitHub will add automatically).
+
+## 8. Connect your production Metabase to your repository
+
+1. Open the Remote sync settings:
+
+   - Click the gear icon in the top right (or navigate to "Admin settings").
+   - In the Settings navigation, click "Remote sync" in the left sidebar.
+
+2. Enter your repository URL:
+
+   - Use the same repository as your development Metabase, for example, `https://github.com/your-org/your-repo.git`.
+
+3. Select Production mode.
+
+4. Add your access token:
+
+   - Paste the read-only personal access token you created for this production Metabase.
+
+5. Save and test the connection:
+
+   - Click "Save changes". Metabase will verify it can reach your repository. If the connection fails, verify your token has the appropriate permissions and hasn't expired.
+
+6. Pull changes and/or enable auto-sync (optional):
+   - Pull changes to sync from your repo.
+   - Toggle on "Auto-sync with Git" to automatically pull changes from your main branch every five minutes.
+
+In Production mode, synced collections appear in the regular collections list with a special icon to indicate that they're versioned and read-only.
+
+At this point, you should be all set up. Exit Admin settings, then reload your browser. You should see your synced Library collection in your production Metabase.
+
+## An example dev-to-production workflow
+
+Let's say your team wants to build a new analytics dashboard. Here's a workflow that ensures that all production content goes through a review process.
+
+### Step 1: Create a new branch
```

Comment:
wait wait wait
branches have not been mentioned in the previous step by step process, so this is kind of out of nowhere